api/maven-api-core/src/main/java/org/apache/maven/api/Constants.java

```diff
@@ -386,5 +386,26 @@ public final class Constants {
     @Config(type = "java.lang.Boolean", defaultValue = "true")
     public static final String MAVEN_CONSUMER_POM = "maven.consumer.pom";
 
+    /**
+     * User property for disabling version resolver cache.
+     *
+     * @since 3.0.0
+     */
+    @Config(type = "java.lang.Boolean", defaultValue = "false")
+    public static final String MAVEN_VERSION_RESOLVER_NO_CACHE = "aether.versionResolver.noCache";
```

Comment:
Maybe we can make it start with `maven.` instead of `aether.` to be consistent with all other properties ?

Reply:
Agreed. 
This component is in maven, but implements Resolver API (as Resolver is "incomplete"), but yes, I agree, this is a Maven implementation detail, not resolver.

Reply:
Fixed